examples/RenderManagerD3DCAPIExample.cpp

```diff
@@ -246,22 +260,23 @@ int main(int argc, char* argv[]) {
     std::vector<ID3D11Texture2D*> depthStencilTextures;
     std::vector<ID3D11DepthStencilView*> depthStencilViews;
 
+	double width = renderInfo[0].viewport.width + renderInfo[1].viewport.width;
+	double height = renderInfo[0].viewport.height;
+
     HRESULT hr;
-    for (size_t i = 0; i < numRenderInfo; i++) {
+    for (size_t i = 0; i < 2; i++) {
```

Comment:
Why make this less general?  A monocular view onscreen is possible when we use numRenderInfo and not when we use 2.

Reply:
The 2 here has a different meaning than the numRenderInfo it replaces. It's no longer the number of render info but the number of buffers in the client-side swap chain, which the client controls and which we only support 1 or 2 at the moment.

For reference, these are the only configurations we support at the moment (i.e. for RegisterRenderBuffers):
1 buffer, false for clientWillNotOverwrite: both eyes in one buffer, single buffered (texture copy)
2 buffers, false for clientWillNotOverwrite: one buffer per eye, single buffered (texture copy)
2 buffers, true for clientWillNotOverwrite: both eyes in one buffer, double buffered (NO texture copy)
4 buffers, true for clientWillNotOverwrite: one buffer per eye, double buffered (NO texture copy)

RenderManager currently has no generic way to support all scenarios and so the client has to make assumptions in order to fit into one of the four supported configurations above - e.g. that there are going to be two render infos, each corresponding to an eye. Ideally, the client would request the suggested viewports and buffer sizes and number for a given client preference (one buffer per eye or all eyes in one buffer X double buffered or not double buffered) and then RenderManager would give the client 1) how many buffers to create, 2) what sizes for each buffer, 3) which RenderInfos to render for which buffers and 4) what viewports on THOSE buffers to use (as opposed to the viewports RenderInfo has, which assumes one buffer per eye).